### DISCUSSION TITLE
Setting for suppressing generation of Virtual interface

### DISCUSSION THREAD

**Discussion by viktornordling** [burningwave/core#8] [Q&A] [answered]
I'm trying to generate an interface to be used by the feign library. For some (kind of silly) reason, it requires that the interface you pass to `feign.Contract.BaseContract#parseAndValidatateMetadata(java.lang.Class<?>)` must only declare one interface, but when I generate an interface with burningwave, it ends up declaring two interfaces: the one I specify and `org.burningwave.core.Virtual`.

Could we make the addition of the `org.burningwave.core.Virtual` optionally disabled, preferably via a call to like `ClassSourceGenerator.addVirtualInterface(false)`, but if that's not wanted a setting in the settings file would be OK also.

**Comment by burningwave** [comment, marked as answer]
It's already present: here an example:
`ClassFactory.ClassRetriever classRetriever = componentSupplier.getClassFactory().loadOrBuildAndDefine(
						LoadOrBuildAndDefineConfig.forUnitSourceGenerator(unitSG).virtualizeClasses(false)
					);`

**Comment by viktornordling** [comment]
Nice, that worked, thanks!
